src/rpc/blockchain.cpp

```diff
@@ -536,10 +536,7 @@ static UniValue getrawmempool(const JSONRPCRequest& request)
                 },
             }.Check(request);
 
-    bool fVerbose = false;
-    if (!request.params[0].isNull())
-        fVerbose = request.params[0].get_bool();
-
+    const bool fVerbose = GetBool(request.params[0], false);
```

Comment:
Doesn't this change behavior when passed an int?

If you want to refactor this to a single line without changing behavior, it would read maybe so:

```suggestion
    const bool fVerbose{request.params[0].isNull() ? false : request.params[0].get_bool()};
```

Reply:
Yeah, sorry, I forgot to comment on that it would allow us to use ints in more cases. I guess the int case is only used in `rawtransaction.cpp` for historical reasons and we rather want to error in other cases? I removed the int part from the function and removed its use from the one place where it's relevant. And I know this can be made a one-liner but I don't like reading ternary operators for the same base case because I still feel like I need to make sure there is not something less intuitive going on like with the `is_witness` stuff, a `!` hiding somewhere etc. But if you don't see the same benefit it's fine if we close it, I think just using ternary operators isn't worth the change.

Reply:
I like the idea of factoring this out to a function, but I don't personally think ints should be accepted where bools are expected in more cases. Being specific about input types helps catch, for example, bugs when the argument is shifted.

Reply:
I agree, I changed that already so there is no behavior change anymore now. I just didn't make a clearer comment on it.